src/main.rs

```diff
@@ -302,15 +312,21 @@ async fn main() {
                 )
                 .await;
                 counter.fetch_add(1, Ordering::Relaxed);
+
+                res
```

Comment:
Return this so that we can accumulate the results once this submission is complete.